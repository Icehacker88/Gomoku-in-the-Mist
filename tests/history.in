place A1
place B3
place D5
history
term
