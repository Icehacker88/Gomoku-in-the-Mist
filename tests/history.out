A1B3D5
